### ISSUE TITLE
Peers should be stored in database, not in configuration

### ISSUE THREAD

**Issue by justmoon** [interledgerjs/ilp-kit#92]
ILP Kit should keep track of current peers as well as adding/removing peers via the database, not via the configuration.

That will allow us to do things like managing peers via the UI (to make it more user-friendly for the operator) and even have an API for adding peers.

**Comment by vhpoet** [comment]
TODO
- [ ] Feed connector with db peers.
- [ ] Remove peers from the config file (ilp-kit-cli).
- [ ] Peer editing.
- [ ] Graphic showing the balance (`----- | --------------`).
- [ ] An invite link for the other side (fills in the form for them).

**Comment by vhpoet** [comment]
@sharafian is there a way to find out who has a one sided peering with you?

**Comment by justmoon** [comment]
@vhpoet: You'll receive RPC messages for a prefix you won't know what to do with...

@sharafian: We could make sure that the first RPC message that plugin-virtual sends is a handshake with the information Vahe needs to detect it as a peering attempt.

@vhpoet: What information should an aspiring peer send? Name, logo, currency?

**Comment by vhpoet** [comment]
@justmoon host and currency for now.

**Comment by justmoon** [comment]
Let's also do limit.

**Comment by vhpoet** [comment]
@justmoon you mean a suggested limit ?

**Comment by justmoon** [comment]
No, I mean the hello packet should include the limit that the other side set, no?

**Comment by justmoon** [comment]
Talked to @vhpoet - the limit can already be queried and there is already code for querying it, so it makes no sense to include in the hello message.

**Comment by michielbdejong** [comment]
Peers are now stored in database, but we still need the 'hello' message.
